### DISCUSSION TITLE
How do I make a scrollable area of very many tiles?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [picoe/Eto#2160] [Q&A]
How do I make a scrollable area of very many tiles? 

I remember the winforms limit of controls is like 10,000 and I can't seem to paint on panel's or scrollables.

**Comment by cwensley** [comment]
Just put a large Drawable inside a Scrollable.  Yes, you'll have a limit of how large that can be on WinForms (no way around that I'm afraid), and on WPF you need to enable tiling via the `Drawable(bool largeCanvas)` constructor, then just handle the `Paint` event to draw the region specified in `PaintEventArgs.ClipRectangle`.
